application/src/sass/cms.scss

```diff
@@ -143,6 +143,15 @@ textarea[disabled] {
     text-decoration: underline;
   }
 
+  /* Table builder */
```

Comment:
I don't think we need any of this new styling.  The new input is currently outside of the form, but if it's moved up inside the form then it should pick up the same `rd-cms.form.input` styling as the rest of the inputs on the page.

Reply:
it isn't a form input to avoid submitting the form on return. we can move it inside the form but will then need to disable it. choice of where to put the messy code. over to you.